api/src/main/resources/org/openmrs/api/db/hibernate/ConceptReferenceRange.hbm.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0"?>
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+
+-->
+<!DOCTYPE hibernate-mapping PUBLIC
+	"-//Hibernate/Hibernate Mapping DTD 3.1//EN"
+	"http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
+<hibernate-mapping package="org.openmrs">
+
+	<class name="ConceptReferenceRange" table="concept_reference_range" batch-size="25">
+
+		<cache usage="read-write"/>
+
+		<id name="conceptReferenceRangeId" type="java.lang.Integer" column="concept_reference_range_id" unsaved-value="undefined">
+			<generator class="identity"/>
+		</id>
+
+		<property name="uuid" type="java.lang.String" column="uuid" length="38" unique="true"/>
+
+		<property name="hiAbsolute" type="java.lang.Double" column="hi_absolute"/>
+		<property name="hiCritical" type="java.lang.Double" column="hi_critical"/>
+		<property name="hiNormal" type="java.lang.Double" column="hi_normal"/>
```

Comment:
Where do we store `units`, `allow_decimal` and `display_precision`?

Reply:
Should we keep them in the `concept_numeric` table and just reference the `concept_numeric_id` foreign key in the `concept_reference_range` table?

Reply:
I think we can just keep them in `concept_numeric` because they are not a hard requirement in `ConceptReferenceRange`, or no? Besides, we have concept_id in `ConceptReferenceRange` so we don't need to reference `concept_numeric`

Reply:
Having a foreign key to the concept_numeric table is different from the one to the concept table because we can have a concept without a corresponding row in the concept numeric table.

Reply:
In that case I can add a reference to `concept_numeric`